Gemfile

```diff
@@ -6,13 +6,13 @@ gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
 gem 'coffee-rails', '~> 4.1.0'
 gem 'jquery-rails'
-gem 'turbolinks'
 gem 'jbuilder', '~> 2.0'
 gem 'sdoc', '~> 0.4.0', group: :doc
 gem 'stripe'
 gem 'activeadmin', '1.0.0.pre2'
 gem 'devise'
 gem 'will_paginate', '~> 3.0.5'
+gem 'money'
```

Comment:
There's another gem called `money_rails` that adds helpers and things, maybe, I'm not sure; I don't know what the difference is, but we use it on weaveup.